src/main/java/svc/data/citations/datasources/importedITI/transformers/ImportedItiCitationTransformer.java

```diff
@@ -0,0 +1,88 @@
+package svc.data.citations.datasources.importedITI.transformers;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import svc.data.citations.datasources.importedITI.models.ImportedItiCitation;
+import svc.data.citations.datasources.transformers.CourtIdTransformer;
+import svc.data.citations.datasources.transformers.MunicipalityIdTransformer;
+import svc.logging.LogSystem;
+import svc.models.Citation;
+import svc.models.Court;
+import svc.types.HashableEntity;
+
+@Component
+public class ImportedItiCitationTransformer {
+
+	@Autowired
+	ImportedItiViolationTransformer violationTransformer;
+
+	@Autowired
+	CourtIdTransformer courtIdTransformer;
+	
+	@Autowired
+	MunicipalityIdTransformer municipalityIdTransformer;
+
+	public List<Citation> fromImportedItiCitations(List<ImportedItiCitation> importedItiCitations) {
+		if (importedItiCitations != null) {
+			return importedItiCitations.stream()
+					.map(importedItiCitation -> fromImportedItiCitation(importedItiCitation))
+					.collect(Collectors.toList());
+		}
+		return null;
```

Comment:
Do we really want to return `null` here or can it be an empty list?

Reply:
See answer below.